spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../../app/models/user'
+
+describe User do
+	
+	let!(:user) do
+		User.create(email: 'test@test.com', username: 'test', password: 'secret1234', password_confirmation: 'secret1234')
```

Comment:
Line is too long. [116/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.